
package msg;

enum type_data
{

// user (11)

    t_sign_up_req = 11101;
    t_sign_up_ack = 11102;

    t_sign_in_req = 11103;
    t_sign_in_ack = 11104;

// character (12) 

    t_create_character_req = 11201;
    t_create_character_ack = 11202;

    t_get_char_list_req = 11203;
    t_get_char_list_ack = 11204;
}
